pathfinder/token_network.py

```diff
@@ -82,24 +77,49 @@ def handle_channel_new_deposit_event(
         """
         Register a new balance for the beneficiary.
 
-        Corresponds to the ChannelNewBalance event. Called by the contract event listener.
+        Corresponds to the ChannelNewDeposit event. Called by the contract event listener.
         """
-        pass
+        assert is_checksum_address(receiver)
+
+        try:
+            participant1, participant2 = self.channel_id_to_addresses[channel_id]
+
+            if receiver == participant1:
+                self.G[participant1][participant2]['view'].update_capacity(deposit=total_deposit)
+            elif receiver == participant2:
+                self.G[participant2][participant1]['view'].update_capacity(deposit=total_deposit)
+            else:
+                log.error(
+                    "Receiver in ChannelNewDeposit does not fit the internal channel"
+                )
+        except KeyError as ke:
+            log.error(
+                "Received ChannelNewDeposit event for unknown channel '{}'".format(
+                    channel_id
+                )
+            )
 
-    def handle_channel_closed_event(self, channel_id: ChannelId):
+    def handle_channel_closed_event(
```

Comment:
why did you change this? There is no reason to split it into multiple lines as long as it fits in one line. Fitting means < `99` columns.